packages/app/components/settings/setting.web.tsx

```diff
@@ -1,33 +1,22 @@
-import { useEffect, useState } from "react";
+import { useState } from "react";
 
-import { useRouter } from "@showtime-xyz/universal.router";
 import { View } from "@showtime-xyz/universal.view";
 
 import { ErrorBoundary } from "app/components/error-boundary";
 import { usePlatformBottomHeight } from "app/hooks/use-platform-bottom-height";
 import { useTabState } from "app/hooks/use-tab-state";
-import { useUser } from "app/hooks/use-user";
 import { WalletAddressesV2 } from "app/types";
 
 import { EditNicknameModal } from "./setting-edit-nickname-moda";
 import { SettingsHeader } from "./setting-header";
 import { SettingTabsScene, SETTINGS_ROUTES } from "./tabs";
 
 const SettingsTabs = () => {
-  const { isAuthenticated } = useUser();
-  const router = useRouter();
   const bottomHeight = usePlatformBottomHeight();
   const [editingWallet, setEditingWallet] = useState<
     WalletAddressesV2 | undefined
   >(undefined);
 
-  useEffect(() => {
```

Comment:
we should avoid using effect for routing, leads to weird bugs sometimes (try logging out from settings after connecting google). Route change can be done in the handler or use some special flag in useEffect if not possible in handler